2019/src/components/landing-page.js

```diff
@@ -25,6 +25,9 @@ const Wrap = styled(Box.withComponent("main"))`
 
 const ImageWrap = styled(Box)`
   display: none;
+  & .gatsby-image-wrapper {
+    position: static !important;
```

Comment:
Not sure the best way to override the image position, but it's necessary so none of the image is cut off in the curve.

Reply:
hmm interesting. this is only temporary, so should be fine 🤷‍♂️